src/session-adapters/cookieAdapter.ts

```diff
@@ -4,37 +4,48 @@ import { Adapter } from './generalAdapter'
 
 const clientSecret = process.env['CLIENT_SECRET'] || ''
 
+const createScopedKey = ({
```

Comment:
I think this is a valid approach we can create, but with the current abstraction we can even recreate the `sessions:[]` structure.

Reply:
Is there any limit to the amount of cookies that can be set per domain?

Reply:
ah you're right. that's true. which implementation do we want?

I believe at least my chrome can support 180 cookies, and 4097 bytes per cookie.

![Screenshot 2024-07-04 at 10 15 38](https://github.com/storyblok/app-extension-auth/assets/499898/e3983ad6-bc27-4f88-a629-95d5c310ce2a)


http://browsercookielimits.iain.guru/

Reply:
Thank you for checking. Just thinking out loud here about the pros and cons: 

**Leaving Session Array:** 
Pros: 

Cons: 
- A single cookie indeed has a rather small limit that can be hit if user has multiple apps on multiple spaces.

**Going with the new approach where each session is its own cookie:**
Pros: 
- Would resolve our previous limitation of a single cookie payload
Cons: 
- Once we update the previous apps, they will be forced to fetch a new session and would leave a stale sb.auth cookie (until it expires of course) 


Anything you would add? So that we can outweight the pros and cons.

Reply:
I think you summarized everything :D 
If we go with the latter, theoretically, we could read the whole cookie and remove expired sessions (it'd require extra work though)